scripts/cloudformation/deploy.sh

```diff
@@ -57,6 +57,12 @@ cd $CLOUDFORMATION_ROOT
 echo "cloudformation auth cloud provider"
 bash $SCRIPTS_DIR/aws/sts.get-caller-identity.sh
 
+# check if required to sync with S3 bucket
+if [ -n "$CFN_TEMPLATE_S3_BUCKET" ] || [ -n "$CFN_S3_PREFIX" ]; then
+  echo "Running S3 Cloudformation Deploy Stack"
+  bash $SCRIPTS_DIR/cloudformation/s3_cloudformation_deploy.sh "$CFN_TEMPLATE_FILENAME" "$CFN_PARAMS_FLAG" "$CFN_TEMPLATE_PARAMS_FILENAME" "$CFN_STACK_NAME" "$CFN_CAPABILITY" "$CFN_TEMPLATE_S3_BUCKET" "$CFN_S3_PREFIX"
+fi
```

Comment:
why is this a separate block than https://github.com/bitovi/bitops/pull/50/files#diff-da6be1326881a745e9220e76c7c0aea95a232cb879d8b6cd13a7452e29ebc584L68
?

As it is, will it not deploy twice if `CFN_STACK_ACTION` is `deploy` AND `CFN_TEMPLATE_S3_BUCKT` is set?

I'd just toss the `CFN_TEMPLATE_S3_BUCKET` and `CFN_S3_PREFIX` to the end of the other deploy script and then handle those variables within the `cloudformation_deploy.sh` script.